util/parseInput.js

```diff
@@ -45,6 +45,8 @@ function fromAny({ first, last, attachments = new Collection() }) {
 		return fromNameAndEmoji(first, last);
 	} else if (!last && attachments.size) {
 		return fromNameAndAttachment(first, attachments.first());
+	} else if (regexes.embedlessUrl.test(last)) {
+		last = last.slice(1, -1);
```

Comment:
This would work on `>add name <url>` input but not `>add <url> name`. Since the current implementation accepts both orders, these lines should be moved up and the `first` variable should go through the same if/slice process

Reply:
What do you mean with that?
> these line should be moved up

Swapping their order seems to work now:
```js
> require('./util/parseInput').fromAny({ first: 'foobar', last: '<https://cdn.discordapp.com/emojis/648531793045356545.png?v=1>' })
{
  name: 'foobar',
  url: 'https://cdn.discordapp.com/emojis/648531793045356545.png?v=1'
}
> require('./util/parseInput').fromAny({ first: '<https://cdn.discordapp.com/emojis/648531793045356545.png?v=1>', last: 'foobar' })
{
  name: 'foobar',
  url: 'https://cdn.discordapp.com/emojis/648531793045356545.png?v=1'
}
```

Reply:
Oh, I was thinking something like this at the very top of the function:
```js
if (regexes.embedlessUrl.test(last)) last = last.slice(1, -1);
else if (regexes.embedlessUrl.test(first)) first = first.slice(1, -1);
// rest of parsing done here
```
But what you did in the latest commit works nicely too.